examples/complete/main.tf

```diff
@@ -10,3 +10,13 @@ module "cloudtrail_s3_bucket" {
 
   context = module.this.context
 }
+
+module "cloudtrail" {
+  source  = "cloudposse/cloudtrail/aws"
```

Comment:
this should be : ```source = "../../"```

Reply:
No, this is a completely separate module.